tecken/settings.py

```diff
@@ -807,42 +807,45 @@ class Prod(Stage):
     """Configuration to be used in prod environment"""
 
 
-class Prodlike(Prod):
```

Comment:
I nixed all the Prodlike stuff a couple of commits ago and missed this configuration.